fresh/utils.q

```diff
@@ -11,6 +11,8 @@ numpy    :.p.import`numpy
 pyStats  :.p.import`scipy.stats
 signal   :.p.import`scipy.signal
 stattools:.p.import`statsmodels.tsa.stattools
+stats_ver:"F"$"." vs (.p.import`statsmodels)[`:__version__]`
+stats_break:$[((stats_ver[0]=0)&stats_ver[1]>=12)|stats_ver[0]>0;1b;0b]
```

Comment:
This can probably be something like
```
stats_ver:"F"$sv["."]2#vs["."].p.import[`statsmodels][`:__version__]`
stats_break:$[0.12<stats_ver;1b;0b]"
```

Reply:
Yes, but technically that would break it if the model version went past
0.99, i know it's unlikely but the above check is meant to catch that as
well.